lib/ain-evm/src/backend.rs

```diff
@@ -195,6 +195,10 @@ impl EVMBackend {
         self.overlay.state.clear()
     }
 
+    pub fn reset(&mut self) {
```

Comment:
```suggestion
    pub fn reset_to_last_changeset(&mut self) 
```
First though on reset is that we reset to initial state

Reply:
Resolved on https://github.com/DeFiCh/ain/pull/2625

Reply:
Will make the switches on https://github.com/DeFiCh/ain/pull/2625

Reply:
Changes are in #2625